### PR TITLE
Fix "public" schema ownership

### PR DESCRIPTION
**Pull request by Sebor**
**Description of the change**

Added additional command
```
```
to fix this [issue](<https://github.com/bitnami/bitnami-docker-postgresql/issues/289>)

**Benefits**

The "public" schema now has right owner

**Possible drawbacks**

-

**Applicable issues**

#289 

**Additional information**

### CODE REVIEW

File: 13/debian-10/rootfs/opt/bitnami/scripts/libpostgresql.sh [hunk 1]
```diff
@@ -466,6 +466,8 @@ postgresql_create_admin_user() {
     echo "CREATE ROLE \"${POSTGRESQL_USERNAME}\" WITH LOGIN ${connlimit_string} CREATEDB PASSWORD '${escaped_password}';" | postgresql_execute
     info "Granting access to \"${POSTGRESQL_USERNAME}\" to the database \"${POSTGRESQL_DATABASE}\""
     echo "GRANT ALL PRIVILEGES ON DATABASE \"${POSTGRESQL_DATABASE}\" TO \"${POSTGRESQL_USERNAME}\"\;" | postgresql_execute "" "postgres" "$POSTGRESQL_PASSWORD"
+    info "Granting access to \"${POSTGRESQL_USERNAME}\" to the 'public' schema in database \"${POSTGRESQL_DATABASE}\""
```
Comment (juan131) [inline 13/debian-10/rootfs/opt/bitnami/scripts/libpostgresql.sh:469]: The message is not quite accurate:
```suggestion
    info "Setting ownership for the 'public' schema database \"${POSTGRESQL_DATABASE}\" to \"${POSTGRESQL_USERNAME}\""
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for the contribution!! Could you please take a look to my comment and replicate the same changes in the branches `9.6`, `10`, `11`, and `12`?

**Comment by Sebor** [comment]
> Thanks so much for the contribution!! Could you please take a look to my comment and replicate the same changes in the branches `9.6`, `10`, `11`, and `12`?

Hi, @juan131!
Done

**Review by juan131 — APPROVED** [review]
Thanks!!! I'm sending your changes for internal review, I'll keep you updated about the progress

**Comment by juan131** [comment]
@Sebor the changes were accepted in the internal review, and new revisions of the PostgreSQL containers (on every branch) will be released within the next 24 hours including them.

Thanks again for this great contribution!

**Pull request merged**